src/SharedPipelineComponents/Datalock/SFA.DAS.CollectionEarnings.DataLock/Domain/LearnerCommitments.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using SFA.DAS.CollectionEarnings.DataLock.Infrastructure.Data.Entities;
+
+namespace SFA.DAS.CollectionEarnings.DataLock.Domain
+{
+    public class LearnerCommitments
+    {
+        public List<Commitment> Commitments { get; }
+        
+        public LearnerCommitments(long key, IEnumerable<CommitmentEntity> commitments)
```

Comment:
key is not used, either remove it or add it as a property. But should be called Uln